pages/boothForm.php

```diff
@@ -0,0 +1,7 @@
+<main>
+  <article class="limit-width-md p-5">
+    <h2 class="article-header">Welcome!</h2>
+    <?php include 'components/divider.php' ?>
+    <p>This page lets you create a booth</p>
```

Comment:
Hey change this maybe?

Reply:
Ok i'll do that, or no i want it this way

Reply:
comment lol